test/utils/urlencode.filter.test.js

```diff
@@ -0,0 +1,13 @@
+describe('Filter:urlencode', function() {
+    var scope = {};
```

Comment:
Technically this scope isn't even needed to be attached to correct? I'm thinking that if someone looked at this for how to use it, they might interpret it that way vs just using the filter/service itself.